community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -62,8 +62,8 @@ variable "instance_image" {
     project = string
   })
   default = {
-    family  = "ubuntu-2204-lts"
-    project = "ubuntu-os-cloud"
+    family  = "debian-12"
```

Comment:
Please include an update to these docs to reflect the new state after this change: https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/docs/vm-images.md#supported-features

Reply:
SG. Nit: I might put a `*` in the untested boxes and say a note below like. "Not yet tested"